cluster/addons/podsecuritypolicies/persistent-volume-binder-role.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: rbac.authorization.k8s.io/v1
+# The persistent volume binder creates recycler pods in the default namespace,
+# but the addon manager only creates namespaced objects in the kube-system
+# namespace, so this is a ClusterRole.
+kind: ClusterRole
+metadata:
+  name: system:podsecuritypolicy:persistent-volume-binder
```

Comment:
typically don't use system prefixes for things created outside the API server / controller manager